ios/utils/RNSScreenModal.h

```diff
@@ -0,0 +1,11 @@
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RNSScreenModal : NSObject
+
++ (BOOL)isFullscreenModal:(UIModalPresentationStyle)presentationStyle;
++ (BOOL)isGrabbableModal:(UIModalPresentationStyle)presentationStyle;
++ (BOOL)isTransparentModal:(UIModalPresentationStyle)presentationStyle;
```

Comment:
I would put these directly on `RNSScreenView` or maybe create a `ModalUtil` category or sth. No need to create separate interface in my opinion.

What do you think?

Reply:
I wouldn't go with category - I've tried that once and it was a bad idea. If I choose to move to the category, I would have to move the whole implementation there to omit circular imports. There I could use methods like `viewDidLayoutSubviews` to use newly-implemented functions there, but I found a strange behavior of modals while doing it - when I was trying to add `viewDidDisappear` to category, after closing the modal it was reappearing again 🤯 after closing it twice, then it disappeared completely.

I'll move them to RNSScreenView 😄